assignment.feature

```diff
@@ -39,7 +39,7 @@ Feature: Plagiarism plugin works with a Moodle Assignment
     And I follow "Test assignment name"
     Then I should see "Grading summary"
 
-  @javascript
+  @javascript @_file_upload
```

Comment:
remove the first underscore

Reply:
The underscore is correct. No idea why but it is.